lib/copyImages.js

```diff
@@ -5,14 +5,22 @@ const cpFile = require('cp-file');
 function copyImages (opts) {
 	return new Promise((resolve, reject) => {
 		// copy images
-		glob(opts.contentsDir + '/**/*.{jpg,png,svg}', (err, files) => {
+		glob(opts.files.contentsDir + '/**/*.{jpg,png,svg}', (err, files) => {
```

Comment:
Should this be `opts.plugins.files`? 

Also, it might make sense to have `copyImages` be part of the `files` plugin?